### PR TITLE
Quick CLI improvements

### PR DESCRIPTION
**Pull request by ctsims**
Minor improvements to the CLI so Matt can more easily fix ICDS issues

X-Requested with: https://github.com/dimagi/commcare/pull/175

### CODE REVIEW

File: util/schema-gen/src/org/javarosa/engine/XFormPlayer.java [hunk 1]
```diff
@@ -461,7 +482,23 @@ private boolean input(String input) throws BadPlaybackException {
...
+                        fec.stepToNextEvent();
+                        return false;
+                    } else {
+                        badInput(input, "Invalid input, only 1 and 2 are allowed");
+                        return false;
+                    }
+                } catch(NumberFormatException nfe) {
+                    badInput(input, "Invalid input");
+                    return false;
+                }
```
Comment (phillipm) [inline util/schema-gen/src/org/javarosa/engine/XFormPlayer.java]: This should be in a helper function

File: util/schema-gen/src/org/javarosa/engine/XFormPlayer.java [hunk 2]
```diff
@@ -159,11 +168,23 @@ private void show(boolean forward) {
...
                 break;
             case FormEntryController.EVENT_PROMPT_NEW_REPEAT:
-                out.print("Repeats Not Implemented, press return to exit");
+                newRepeatQuestion();
                 break;
         }
     }
 
+    private void newRepeatQuestion() {
+        String repeatType = getCurrentGroupName();
```
Comment (phillipm) [inline util/schema-gen/src/org/javarosa/engine/XFormPlayer.java]: Unused

File: util/schema-gen/src/org/javarosa/engine/XFormPlayer.java [hunk 3]
```diff
@@ -159,11 +168,23 @@ private void show(boolean forward) {
...
+        out.println("Add new repeat?");
+        out.println("1) Yes, add a new repeat group");
+        out.println("2) No, continue to the next question");
+    }
+
+    private String getCurrentGroupName() {
```
Comment (phillipm) [inline util/schema-gen/src/org/javarosa/engine/XFormPlayer.java]: Will be unused once you remove the call above

**Comment by ctsims** [comment]
@phillipm updated with PR feedbck

**Pull request merged**